crm_phonecall/report/crm_phonecall_report.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+# Copyright 2004-2010 Tiny SPRL (<http://tiny.be>)
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import tools
+from openerp import fields, models
+
+AVAILABLE_STATES = [
```

Comment:
Why don't you take it from the same source instead of creating the list 2 times?